src/models/schemas/user-schema.js

```diff
@@ -27,6 +27,10 @@ const UserSchema = new Schema({
     country: String,
     city: String,
   },
+  shipping_address: {
```

Comment:
camel case

Reply:
also it can have address related info too,

need to check body validation when it is added to see what fields there are